dot/parachain/types/statement_test.go

```diff
@@ -160,11 +160,11 @@ func TestStatementVDT_SignAndVerify(t *testing.T) {
 	require.NoError(t, err)
 
 	publicKeyBytes := keyPair.Public().Encode()
-	validatorID := ValidatorID(publicKeyBytes)
+	ValidatorPublicKey := ValidatorPublicKey(publicKeyBytes)
```

Comment:
same as above, Let's make the variable unexported
```suggestion
	validatorPublicKey := ValidatorPublicKey(publicKeyBytes)
```